src/wdio/services/index.js

```diff
@@ -1,11 +1,9 @@
 import AxeService from './AxeService';
 import TerraService from './TerraService';
 import SeleniumDockerService from './SeleniumDockerService';
-import WebpackDevServerService from './WebpackDevServerService';
-import ExpressDevServerService from './ExpressDevServerService';
+import TerraToolkitServeStaticService from './TerraToolkitServeStaticService';
```

Comment:
This feels overlay robust. Can we shorten to `ServeStaticService`?

Reply:
I kind of like identifying as a terra-toolkit service, but I'm open to changing if others feel strongly.

Reply:
I'd prefer without. 2 reasons, 1 - it might be confusing since we also offer the `TerraService` 2- the toolkit offers several custom services and if we change we should rename all to be prepended with 'TerraToolkit'